.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "C:\\Users\\jjakubi\\AppData\\Local\\Programs\\Python\\Python37\\python.exe"
+    "python.pythonPath": "C:\\Users\\AMinavi1\\AppData\\Local\\Programs\\Python\\Python37-32\\python.exe"
 }
```

Comment:
We'll need to remove this file from getting checked in again and again... I understand it might not be straightforward to you, so let's plan on working on this together in person in tonight's class. I'll find you in class. Thanks.